.github/PULL_REQUEST_TEMPLATE/new_icon.md

```diff
@@ -7,3 +7,7 @@
 - [] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)
 
 Refer to the [`CONTRIBUTING.md`](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) for more details.
+
+
+**Link to official page to prove your SVG is correct and update to date**
+*Link goes here*
```

Comment:
Add a blank line between the headline and the actual link to have them separated like the other headlines (see the preview).

```suggestion
**Link to official page to prove your SVG is correct and update to date**

*Link goes here*
```